scapy/contrib/mount.py

```diff
@@ -0,0 +1,143 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Lucas Preston <lucas.preston@infinite.io>
+# This program is published under a GPLv2 license
+
+# scapy.contrib.description = NFS Mount v3
+# scapy.contrib.status = loads
+
+from binascii import unhexlify
+import scapy.contrib.oncrpc as rpc
+from scapy.packet import Packet, bind_layers
+from scapy.fields import IntField, StrLenField, IntEnumField, PacketField, \
+    ConditionalField, FieldListField
+
+mountstat3 = {
+    0: 'MNT3_OK',
+    1: 'MNT3ERR_PERM',
+    2: 'MNT3ERR_NOENT',
+    5: 'MNT3ERR_IO',
+    13: 'MNT3ERR_ACCES',
+    20: 'MNT3ERR_NOTDIR',
+    22: 'MNT3ERR_INVAL',
+    63: 'MNT3ERR_NAMETOOLONG',
+    10004: 'MNT3ERR_NOTSUPP',
+    10006: 'MNT3ERR_SERVERFAULT'
+}
+
+
+class Path(Packet):
+    name = 'Path'
+    fields_desc = [
+        IntField('length', 0),
+        StrLenField('path', '', length_from=lambda pkt: pkt.length),
+        StrLenField('fill', '', length_from=lambda pkt: (4 - pkt.length) % 4)
+    ]
+
+    def extract_padding(self, s):
+        return '', s
+
+    def set(self, path, length=None, fill=None):
+        if length is None:
+            length = len(path)
+        if fill is None:
+            fill = '\x00' * ((4 - len(path)) % 4)
+        self.length = length
+        self.path = path
+        self.fill = fill
+
+
+class Filehandle(Packet):
```

Comment:
This is a duplicate of `File_Object`. Could you either:
- make one class inherit the other (then change the name)
- make one class have
```python
fields_desc = the_other.fields_desc
set=the_other.set
```